README.md

```diff
@@ -149,14 +149,14 @@ else
 If you want to use TzClock you have to set -DLIBCRON_BUILD_TZ_CLOCK=ON when building libcron. TzClock is a fully optional feature
 if you don't enable it, it won't be build at all.
 
-Using TzClock has the following side effects:
-  1. Date requires linkage to `libcurl`
-  2. First time TzClocks timezone lookup will occur it will download the most up to date timezone: "~/Downloads/tzdata" ("%homedrive%\%homepath%\downloads\tzdata" on Windows).
-  3. TzClock uses a `std::mutex` to protect the time zone in multithreaded envoirements.
-  4. This implementation might decrease performance a lot based on point 2 and 3.
+By default when setting the time zone with `TzClock::set_time_zone`, date-tz will automatically download the neweset database version from [Time Zones](https://www.iana.org/time-zones).
+This can be disabled with `-DLIBCRON_MANUAL_TZ_DB=ON`. If this is set user will be responsible of downloading timezone database.
 
-[More Info about date-tz](https://howardhinnant.github.io/date/tz.html)
+Setting database path / install path can be done with `date::set_install`. If not set path will default to: "~/Downloads/tzdata" ("%homedrive%\%homepath%\downloads\tzdata" on Windows).
+Using TzClock without LIBCRON_MANUAL_TZ_DB requires liburl installed in your system. On Windows it will also require 7-Zip installed into the default location.
```

Comment:
Wait, it requires 7-zip ? Meaning it will execute it to unpack? I'll have to pull the emergency break here. This library should not execute any application, too much of a security issue the user of this library doesn't have control over.

With that in mind, we'll have to settle with using a pre-existing DB and let the library user handle management of that file.

Reply:
With that in mind, the TzClock can take a path to the DB as a constructor argument and call date:set_install, ensuring it is available before the construction of the clock.